### ISSUE TITLE
Need the option to mask the input and output of the LLM API in Datadog LLM observability.

### ISSUE THREAD

**Issue by Gekko0114** [DataDog/dd-trace-py#10517]
### Summary of problem

Since the input and output of the LLM API contain sensitive information, I don't want to send them to Datadog.
I would like an option to send data to Datadog with the input and output masked.
(If it's okay, I can create a PR for this.)
Or, if it is already implemented, please let me know.

**Comment by Yun-Kim** [comment]
Hi @Gekko0114, are you using manual instrumentation? Or relying only on the OpenAI/LangChain/Bedrock/Anthropic integrations?

**Comment by Gekko0114** [comment]
I am relying on openai and other APIs.

**Comment by Yun-Kim** [comment]
Would be able to tell us some more details of your use case with LLM Observability? Obfuscating input/output will significantly impact many of the features that LLM Observability offers. APM (and not LLM Observability) might fit your use case better here.

**Comment by Gekko0114** [comment]
Hi @Yun-Kim 
My goal is to obtain latency, token counts, and other information when calling LLM API requests.
If I want to gather these metrics using APM, what steps should I take?
(I already enabled APM on my app)

BTW, I am using LLM obs like below.
```python
with LLMObs.llm(
        model_name=model,
        name=name,
        session_id=session_id
    ) as span:
    call_api()
    LLMObs.annotate(span=span, input_data=input, output_data=output)
```

**Comment by Yun-Kim** [comment]
Which LLM APIs are you using apart from OpenAI? If you are using only our integrations (OpenAI/Bedrock/LangChain/Anthropic), you can continue using those APM integrations but set `DD_{INTEGRATION_NAME}_SPAN_PROMPT_COMPLETION_SAMPLE_RATE=0` to disable capturing input/outputs. You will not need LLM Observability since the APM integrations should be more compatible for your use case.

If you are using LLM providers we currently don't provide integrations for, you can continue using LLM Observability as you've specified, you just need to omit annotating the span with `input_data/output_data` arguments. However, it should be noted that omitting input/output data in LLM spans will impact some LLM Observability features.

**Comment by Gekko0114** [comment]
Thanks @Yun-Kim, I will try it.
BTW, do you plan on preparing an integration for Gemini API? It would be helpful for me.

**Comment by Yun-Kim** [comment]
@Gekko0114 [gemini tracing support](https://github.com/DataDog/dd-trace-py/pull/10503) has been merged to main, we are waiting on it to be released as part of `ddtrace==2.14` 😄 

I'll close this in the meantime, feel free to reopen/open another issue if you have any problems!
